x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -129,6 +129,10 @@ const RuleTypeEuiFormRow = styled(EuiFormRow).attrs<{ $isVisible: boolean }>(({
   },
 }))<{ $isVisible: boolean }>``;
 
+const IntendedRuleTypeEuiFormRow = styled(RuleTypeEuiFormRow)`
+  ${({ theme }) => `padding-left: ${theme.eui.euiSizeXL};`}
+`;
+
```

Comment:
I heard in passing that we're encouraged to use `emotion` over `styled-components` these days. I honestly prefer the `style-components` way you're using here by declaring a new semantic 'component', but figured I'd bring it up in case others know more or could point to relevant resources, as we currently have conflicting information in our dev docs [here](https://docs.elastic.dev/security-solution/adaptive-workload-protection/platform/visualization-team/guides/pr-review-checklist#awp-viz-team-style-guide) & [here](https://docs.elastic.dev/kibana-dev-docs/contributing/styleguide#sass-files). 🤷‍♂️

Reply:
tried first `emotion`, but it wasn't working. That's how I discovered [issues and workaround related to CSS-in-JS, when it styled-components conflict with emotion](https://elastic.slack.com/archives/G01CTGPURDK/p1683307739315749 ), when speaking with `eui` team up.
I decided then to leave it as it was implemented, to not mix for one component both css-in-js and styled components